test/e2e/tests/request-queuing/multiple-networks-dapps-txs.spec.js

```diff
@@ -23,8 +25,8 @@ describe('Request Queuing for Multiple Dapps and Txs on different networks.', fu
         dapp: true,
         fixtures: new FixtureBuilder()
           .withNetworkControllerDoubleGanache()
-          .withPermissionControllerConnectedToTwoTestDapps()
-          .withSelectedNetworkControllerPerDomain()
+          // .withPermissionControllerConnectedToTwoTestDapps()
+          // .withSelectedNetworkControllerPerDomain() // TODO: Add back when refreshing state doesn't revert per dapp selected network. Remove having to connect to dapp
```

Comment:
Hmm, what does this comment refer to? I'm not familiar with this problem

Reply:
I'm not totally understanding this either. Currently working with Thomas to figure it out!

Reply:
ok this was a misunderstanding. I worked through it with @tmashuang  and we've updated the e2e tests here https://github.com/MetaMask/metamask-extension/pull/23130/commits/85a1d28989f7f61472c0ad65a787a98023957752 so that they work as expected now. Let me know if you have any further questions @Gudahtt